static/js/pages/certificates/CertificateGenerate.tsx

```diff
@@ -121,6 +121,15 @@ const CertificateGenerate: FC = () => {
             <h3 className="p-stepped-list__title">
               All done <Icon name="success" />
             </h3>
+            <div className="p-stepped-list__content">
+              <Row>
+                <Col size={8}>
+                  <p>
+                    <a href="/">Go back to the main page</a>.
```

Comment:
Does it have to be an `a`? With page reloading?

Reply:
You mean with `useNavigate`? Or, can you give me a quick code example of what you mean?

Reply:
I mean using `<Link to="/">Go back to the main page</Link>` from `react-router-dom` instead of `a`. If you use a HTML anchor tag it will always reload the page and lose your state.

Reply:
Ah, yes, now I understand your question. Yes, I need to lose my state here because I need to check again if the user is now (= after following the steps for certificate generation and whitelisting) authenticated or not - if I don't, the user will see the `NoAuth` page forever until they reload the page. So yes, in short, a page reload is what we want here.